.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -29,7 +31,10 @@ Library version: x.x.x
 ## Steps To Reproduce
 
 <!--
- Issues without reproduction steps or code are likely to stall.
+- You must provide an easy way to reproduce the problem.
+- Keep the code reproducing the bug as simple as possible, with the minimum amount of code required to reproduce the issue. [Read more](https://stackoverflow.com/help/mcve).
```

Comment:
Does this render as link to the user? I have never seen this being used.

If not, it's better to remove the `[Read more]` portion.

Reply:
right; I did what you suggested